src/mame/mame.lst

```diff
@@ -1427,6 +1427,7 @@ atarifb1                        // 033xxx           1978/10 [6502]
 atarifb2                        // 035330-035350    ???     [6502]
 atarifb4                        // 034754           1979/04 [6502]
 soccer                          // 035222-035260    1980/04 [6502]
+amerug                          // 1980 Shoei
```

Comment:
Please order the systems by short name here.

Reply:
Does this mean reordering like this?
atarifb2                        // 035330-035350    ???     [6502]
atarifb4                        // 034754           1979/04 [6502]
amerug                          // 1980 Shoei
soccer                          // 035222-035260    1980/04 [6502]

Or should it be in the same order as atari/atarifb.cpp?
atarifb2                        // 035330-035350    ???     [6502]
amerug                          // 1980 Shoei
atarifb4                        // 034754           1979/04 [6502]
soccer                          // 035222-035260    1980/04 [6502]

Reply:
mame.lst is ordered by alphabet